stories/atoms/Map.tsx

```diff
@@ -91,14 +91,19 @@ export default {
     },
 };
 
-export const _Map = ({ width, height, ...args }) => (
+type MapArguments = {
+    width: string;
+    height: string;
+} & MapProps;
+
+export const _Map = ({ width, height, ...mapProps }: MapArguments) => (
```

Comment:
why to rename `args`? 
`args` is what we call it in all the stories and it easy to understand that they are coming from the `_Map.args` object below. I would keep it consistent

Reply:
I don't mind changing, but personally, I find using `...mapProps` and `...props` to be more explicit in conveying the purpose of these variables.